tests/php/api/controller/class-test-audit-posts-controller.php

```diff
@@ -14,15 +14,6 @@
  */
 class Test_Audit_Posts_Controller extends WP_Test_REST_Controller_TestCase {
 
-	/**
-	 * REST Server.
-	 *
-	 * Note that this variable is already defined on the parent class but it lacks the phpdoc variable type.
-	 *
-	 * @var WP_REST_Server
-	 */
-	protected $server;
-
```

Comment:
Why remove this and not set the server in the setUp method?

Reply:
Basically all of the changes to the code below are not needed if the setUp method is changed